certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+{
+  "x-generator": "NSwag v13.14.8.0 (NJsonSchema v10.6.1.0 (Newtonsoft.Json v13.0.0.0))",
```

Comment:
@aiforged Please remove this property as it is not required and supported by our system, it may fail your connector in the validation phase in the certification process.

Reply:
ok, will remove - but the paconn did not complain about this